test/csuite/timestamp_abort/main.c

```diff
@@ -810,6 +859,11 @@ thread_backup_run(void *arg)
         sleep_time = __wt_random(&td->extra_rnd) % MAX_BACKUP_INVL;
         __wt_sleep(sleep_time, 0);
 
+        if (backup_test_force_stop && i % 2 == 0) {
```

Comment:
This definitely feels too frequent and makes successive incremental backups not happen. Having something hard-coded here makes `backup_full_interval` something that isn't useful any longer. This should be tied to the full backup interval. Maybe every other one?

However, I've seen that often times the `backup_full_interval` is never triggered. If I'm running with it set to, say, 4, and multiple iterations, the counter is reset in between each crash/restart. So if we have 3 incremental backups per "kill the child" period, the next run will continue to do incremental. We may get 8 or 9 incremental backups in a row instead of ever restarting with a full backup in there.

Reply:
Alternatively, we can invoke this with a certain probability (like 20% or 25%), irrespective of the cycle, so that we can cover different cases over time. And if we call it within the first cycle, it would have the benefit of forcing next full backup early.

To make it more likely for us to get through at least one full cycle, we can either increase the default length of the test in smoke.sh from 10s to 15 or even 20s, or we can maybe just turn off `backup_verify_immediately`, so that there is less delay, and we can squeeze more into those 10s. I feel like this option is great when debugging something, but I'm not sure it's necessary to be on during normal testing, as it just makes the test fail earlier. That being said, that option may be nice for hard-to-reproduce bugs—but let me know what you think.